core/src/main/java/edu/wpi/grip/Main.java

```diff
@@ -11,46 +12,42 @@
 import edu.wpi.grip.core.operations.opencv.NewSizeOperation;
 import edu.wpi.grip.core.sinks.DummySink;
 import edu.wpi.grip.generated.CVOperations;
-import edu.wpi.grip.ui.pipeline.PipelineView;
-import edu.wpi.grip.ui.preview.PreviewsView;
-import javafx.fxml.FXML;
-import javafx.fxml.FXMLLoader;
-import javafx.scene.control.ScrollPane;
-import javafx.scene.control.SplitPane;
-import javafx.scene.layout.VBox;
 
-import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
 
-/**
- * The Controller for the application window.
- */
-public class MainWindowView extends VBox {
+public class Main {
+    private static final EventBus eventBus;
+    private static final Core core;
 
-    @FXML
-    private SplitPane topPane;
+    static {
+        eventBus = new EventBus((exception, context) -> {
+            Main.eventBus.post(new FatalErrorEvent(exception));
+        });
+        core = new Core(eventBus, new Pipeline(eventBus), () -> loadAllOperations());
+    }
 
-    @FXML
-    private ScrollPane bottomPane;
+    public static class Core {
+        public final EventBus eventBus;
+        public final Pipeline pipeline;
+        public final Runnable loadOperations;
 
-    public MainWindowView(final EventBus eventBus) {
-        eventBus.register(this);
-        try {
-            FXMLLoader fxmlLoader = new FXMLLoader(getClass().getResource("MainWindow.fxml"));
-            fxmlLoader.setRoot(this);
-            fxmlLoader.setController(this);
-            fxmlLoader.load();
-        } catch (IOException e) {
-            throw new RuntimeException("FXML Failed to load", e);
+        private Core(final EventBus eventBus, final Pipeline pipeline, final Runnable loadOperations) {
+            this.eventBus = eventBus;
+            this.pipeline = pipeline;
+            this.loadOperations = loadOperations;
         }
+    }
 
-        final PreviewsView previewsView = new PreviewsView(eventBus);
-        final PaletteView paletteView = new PaletteView(eventBus);
-        final PipelineView pipelineView = new PipelineView(eventBus, new Pipeline(eventBus));
+    protected static Core getCore() {
```

Comment:
Singletons aren't cool.  This will prevent us from ever having multiple projects open